x-pack/plugins/security_solution/public/common/containers/source/mock.ts

```diff
@@ -469,6 +469,30 @@ export const mockBrowserFields: BrowserFields = {
         type: 'date',
         readFromDocValues: true,
       },
+      _id: {
```

Comment:
The `browserFields` mocks were inconsistent with the `defaultHeaders` mocks, on both securitySolution and timelines. In order to test the new changes properly, I had to make both mocks consistent (without any missing field). This led me to update a bunch of unrelated tests and snapshots.